interface/patient_file/report/patient_report.php

```diff
@@ -86,456 +86,462 @@ function show_date_fun(){
-                    </td>
-                </tr>
-            </table>
-            <button type="button" class="genreport btn btn-secondary btn-save btn-sm" value="<?php echo xla('Generate Report'); ?>" ><?php echo xlt('Generate Report'); ?></button>
-            <button type="button" class="genpdfrep btn btn-secondary btn-download btn-sm" value="<?php echo xla('Download PDF'); ?>" ><?php echo xlt('Download PDF'); ?></button>
-
-            <!-- Procedure Orders -->
-            <hr/>
-            <table class="border-0" cellpadding="0" cellspacing="0">
-                <tr>
-                    <td class='font-weight-bold'><span class='oe-report-section-header'><?php echo xlt('Procedures'); ?>:</span></td>
-                    <td class='text'>&nbsp;<?php echo xlt('Order Date'); ?>&nbsp;&nbsp;</td>
-                    <td class='text'><?php echo xlt('Encounter Date'); ?>&nbsp;&nbsp;</td>
-                    <td class='text'><?php echo xlt('Order Descriptions'); ?></td>
-                </tr>
-                <?php
-                $res = sqlStatement(
-                    "SELECT po.procedure_order_id, po.date_ordered, fe.date " .
-                    "FROM procedure_order AS po " .
-                    "LEFT JOIN forms AS f ON f.pid = po.patient_id AND f.formdir = 'procedure_order' AND " .
-                    "f.form_id = po.procedure_order_id AND f.deleted = 0 " .
-                    "LEFT JOIN form_encounter AS fe ON fe.pid = f.pid AND fe.encounter = f.encounter " .
-                    "WHERE po.patient_id = ? " .
-                    "ORDER BY po.date_ordered DESC, po.procedure_order_id DESC",
-                    array($pid)
-                );
-                while ($row = sqlFetchArray($res)) {
-                    $poid = $row['procedure_order_id'];
-                    echo " <tr>\n";
-                    echo "  <td align='center' class='text'>" .
-                    "<input type='checkbox' name='procedures[]' value='" . attr($poid) . "' />&nbsp;&nbsp;</td>\n";
-                    echo "  <td class='text'>" . text(oeFormatShortDate($row['date_ordered'])) . "&nbsp;&nbsp;</td>\n";
-                    echo "  <td class='text'>" . text(oeFormatShortDate($row['date'])) . "&nbsp;&nbsp;</td>\n";
-                    echo "  <td class='text'>";
-                    $opres = sqlStatement(
-                        "SELECT procedure_code, procedure_name FROM procedure_order_code " .
-                        "WHERE procedure_order_id = ? ORDER BY procedure_order_seq",
-                        array($poid)
-                    );
-                    while ($oprow = sqlFetchArray($opres)) {
-                        $tmp = $oprow['procedure_name'];
-                        if (empty($tmp)) {
-                            $tmp = $oprow['procedure_code'];
+                    <span class="bold oe-report-section-header"><?php echo xlt('Documents'); ?>:</span><br />
+                    <ul>
```

Comment:
Hi @tywrenn 

Such as this 'bold' class, should I convert it into the `font-weight-bold`?

Reply:
Yeah the bold class is sooooooooo deprecated